.github/workflows/nodejs.yml

```diff
@@ -17,15 +17,76 @@ jobs:
     name: test
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        node-version:
-          - 16.x
-          - 18.x
-          # TODO: Resolve whttps://github.com/LavaMoat/LavaMoat/issues/1211 and enable v20.x, v22.x
-          - 20.14.x
-          - 22.5.x
-          #- 20.x
-          #- 22.x
+        inlude:
+          - node-version: 16.x
+            test-workspaces: >-
+              --workspace=packages/aa
+              --workspace=packages/allow-scripts
+              --workspace=packages/lavapack
+              --workspace=packages/preinstall-always-fail
+              --workspace=packages/tofu
```

Comment:
I'm not sure why lavapack and tofu are on this list. Understand the rest.

Reply:
Sure, follow-up is fine - especially that it'd be considered a breaking change. I am not aware of them being dependencies of any packages where we intentionally keep node 16 support.